File: app/src/main/java/io/treehouses/remote/pojo/NetworkListItem.java
package io.treehouses.remote.pojo;

import java.util.ArrayList;
import java.util.List;

import io.treehouses.remote.R;

public class NetworkListItem {
    private String title;
    private int layout;

    public NetworkListItem(String title, int layout) {
        this.title = title;
        this.layout = layout;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public int getLayout() {
        return layout;
    }

    public void setLayout(int layout) {
        this.layout = layout;
    }

    public static List<NetworkListItem> getSystemList() {
        List<NetworkListItem> systemList = new ArrayList();
        systemList.add(new NetworkListItem("Open VNC", R.layout.open_vnc));
        systemList.add(new NetworkListItem("Configure Tethering (beta)", R.layout.configure_tethering));
        systemList.add(new NetworkListItem("Add SSH Key", R.layout.configure_ssh_key));
        systemList.add(new NetworkListItem("Toggle Camera", R.layout.configure_camera));
        systemList.add(new NetworkListItem("Wifi Country", R.layout.configure_wificountry));
        systemList.add(new NetworkListItem("Blocker Level", R.layout.configure_blocker));
        systemList.add(new NetworkListItem("Discover", R.layout.configure_discover));
        return systemList;
    }
}

File: app/src/main/java/io/treehouses/remote/pojo/NetworkListItem.kt
package io.treehouses.remote.pojo

import io.treehouses.remote.R
import java.util.*

class NetworkListItem(var title: String, var layout: Int) {

    companion object {
        val networkList: List<NetworkListItem>
            get() {
                val list: MutableList<NetworkListItem> = ArrayList()
                list.add(NetworkListItem("Ethernet: Automatic", R.layout.dialog_ethernet))
                list.add(NetworkListItem("WiFi", R.layout.dialog_wifi))
                list.add(NetworkListItem("Hotspot", R.layout.dialog_hotspot))
                list.add(NetworkListItem("Bridge", R.layout.dialog_bridge))
                list.add(NetworkListItem("Reset", R.layout.dialog_reset))
                list.add(NetworkListItem("Reboot", R.layout.dialog_reboot))
                list.add(NetworkListItem("Network Mode: ", -1))
                return list
            }

        val systemList: List<NetworkListItem?>
            get() {
                val systemList: MutableList<NetworkListItem?> = ArrayList<Any?>()
                systemList.add(NetworkListItem("Open VNC", R.layout.open_vnc))
                systemList.add(NetworkListItem("Configure Tethering (beta)", R.layout.configure_tethering))
                return systemList
            }
    }

}